crates/dies-world/src/filter/filter_builder.rs

```diff
@@ -0,0 +1,46 @@
+use nalgebra as na;
+use na::{ OMatrix, U2,};
+use na::OVector;
+use na::U4;
+use crate::filter::kalman::Kalman;
+use crate::filter::matrix_gen::{ULMotionModel, WhiteNoise1stOrder};
+
+pub(crate) struct KalmanBuilder {
```

Comment:
Is a separate builder type needed? Could this just be moved to the constructor of `Kalman`?